src/components/NavBar.vue

```diff
@@ -198,7 +199,6 @@ export default {
       v-model="connectionMenu"
       :close-on-content-click="false"
       offset-y
-      open-on-hover
```

Comment:
I actually like the open-on-hover - not sure I'd realize it was clickable and so might miss all the extra useful info.

Reply:
I'm thinking the consistency across the nav bar is better in this case, since all the other buttons on the top nav are on-click instead of on-hover. Perhaps a middle ground (but out of scope of this PR) would be to add tooltips to all the navbar buttons so it's easy to understand what they're meant for?